content/en/observability_pipelines/setup/_index.md

```diff
@@ -5,8 +5,399 @@ type: multi-code-lang
 aliases:
   - /getting_started/observability_pipelines/
   - /observability_pipelines/installation/
+further_reading:
+  - link: "/observability_pipelines/working_with_data/"
+    tag: "Documentation"
+    text: "Working with data in Observability Pipelines"
+  - link: "/observability_pipelines/production_deployment_overview/"
+    tag: "Documentation"
+    text: "Deployment Design and Principles for the Observability Pipelines Worker"
+  - link: "/observability_pipelines/architecture/"
+    tag: "Documentation"
+    text: "Production deployment design and principles for the Observability Pipelines Worker"
+  - link: "https://dtdg.co/d22op"
+    tag: "Learning Center"
+    text: "Safe and Secure Local Processing with Observability Pipelines"
 ---
 
-For setup instructions, select your use case:
+## Overview
+
+The [Observability Pipelines Worker][1] can collect, process, and route logs and metrics from any source to any destination. Using Datadog, you can build and manage all of your Observability Pipelines Worker deployments at scale.
+
+There are several ways to get started with the Observability Pipelines Worker.
+
+- [Quickstart](#quickstart): Install the Worker and test out the functionality of Observability Pipelines with demo data.
+- [Datadog setup guide][2]: Install the Worker and connect it to Datadog.
+- [Splunk setup guide][3]: Install the Worker and connect it to Splunk.
+
+This document walks you through the quickstart installation steps and then provides resources for next steps.
+
+## Prerequisites
+
+To install the Observability Pipelines Worker, you need the following:
+
+- A valid [Datadog API key][4].
+- A pipeline ID.
+
+To generate a new API key and pipeline:
+
+1. Navigate to [Observability Pipelines][5].
+2. Click **New Pipeline**.
+3. Enter a name for your pipeline.
+4. Click **Next**.
+4. Select the template you want and follow the instructions.
+
+## Quickstart
+
+Follow the below instructions to install the Worker and deploy a sample pipeline configuration that uses demo data.
+
+### Install the Observability Pipelines Worker
+
+{{< tabs >}}
+{{% tab "Docker" %}}
+
+The Observability Pipelines Worker Docker image is published to Docker Hub [here][1].
+
+1. Download the [sample pipeline configuration file][2] that uses demo data for this quickstart. See [Configurations][3] for more information about the source, transform, and sink used in the sample configuration.
```

Comment:
"1. Download the [sample pipeline configuration file][2] to quickly get started. This configuration will emit demo data, parse and structure the data, and emit them to console and Datadog. See [Configurations][3] for more information about the source, transform, and sink used in the sample configuration."